docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -639,14 +639,14 @@ The default configuration files are installed at:
 
 ??? example "Filtering devices using snap options"
```

Comment:
do we normally do `???` to start examples or is it `!!!`

Reply:
`???` renders it collapsed. REader has to expand. Never use it as I don't like it collapsed.